.circleci/config.yml

```diff
@@ -2,28 +2,28 @@ version: 2
 jobs:
   license:
     docker:
-      - image: golang:1.19
+      - image: golang:1.21
     steps:
       - checkout
       - run: ./script/check-licenses.sh
   vet:
     docker:
-      - image: golang:1.19
+      - image: golang:1.21
     steps:
       - checkout
       - run: go vet ./...
   test:
     docker:
-      - image: golang:1.19
+      - image: golang:1.21
     steps:
       - checkout
       - run: go test -race -cover ./...
   smoke-test:
     docker:
-      - image: golang:1.19
+      - image: golang:1.21
     steps:
       - checkout
-      - run: go run ./cmd/clusterlint --version
```

Comment:
Had to fix this since the `--version` flag is not exposed when the Version is empty in the struct after the cli upgrade to `v1.22.14`. Currently, we set it manually when doing the `go build` command but you can't specify the same flag on `go run`. That said, I refactored it to use the `--help` flag.

Ref: https://github.com/urfave/cli/pull/954/files